manifests/base/tasks/secret.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I am not sure why this secret is present here... Is it meant to be for dev environment? If so it should be present locally if it is supposed to be in staging environment or production it should be in that overlay. Also you should not have secrets in this opened form. For this situation we are using [Ksops](https://github.com/viaduct-ai/kustomize-sops) How file looks after using ksops you can see [here](https://github.com/operate-first/peribolos-as-a-service/blob/main/manifests/overlays/stage/controller-secret.enc.yaml)
If especially `private_key` is currently in use somewhere please regenerate it.

Reply:
Will get on that this week. What would be the best way to contact you if I have any questions?

Reply:
Great!, propably best for start will be if you could join our [slack]( https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww)

Reply:
Another option would be to use vault. Regardless of how the secret is stored you will need to regenerate your credentials for your GitHub application, as this is considered a leak.